modules/osd-create-cluster-gcp-account.adoc

```diff
@@ -64,15 +64,18 @@ With *Use Custom KMS keys* selected:
 ... Provide the *KMS Service Account*.
 
 +
-.. Optional: Select *Enable additional etcd encryption* if you require etcd key value encryption.
-With this option, the etcd key values are encrypted, but not the keys. This option is in addition to the control plane storage encryption that encrypts the etcd volumes in {product-title} clusters by default.
+.. Optional: Select *Enable FIPS cryptography* if you require your cluster to be FIPS validated.
 +
 [NOTE]
 ====
-By enabling etcd encryption for the key values in etcd, you incur a performance overhead of approximately 20%. The overhead is a result of introducing this second layer of encryption, in addition to the default control plane storage encryption that encrypts the etcd volumes. Consider enabling etcd encryption only if you specifically require it for your use case.
+If  *Enable FIPS cryptography* is selected, *Enable additional etcd encryption* is enabled by default and cannot be disabled. Users can choose to select *Enable additional etcd encryption* without selecting *Enable FIPS cryptography*.
 ====
+.. Optional: Select *Enable additional etcd encryption* if you require etcd key value encryption. With this option, the etcd key values are encrypted, but not the keys. This option is in addition to the control plane storage encryption that encrypts the etcd volumes in {product-title} clusters by default.
 +
-.. Optional: Select *Enable FIPS cryptography* if you require your cluster to be FIPS validated.
+[NOTE]
+====
+By enabling etcd encryption for the key values in etcd, you will incur a performance overhead of approximately 20%. The overhead is a result of introducing this second layer of encryption, in addition to the default control plane storage encryption that encrypts the etcd volumes. Consider enabling etcd encryption only if you specifically require it for your use case.
```

Comment:
```suggestion
By enabling etcd encryption for the key values in etcd, you incur a performance overhead of approximately 20%. The overhead is a result of introducing this second layer of encryption, in addition to the default control plane storage encryption that encrypts the etcd volumes. Consider enabling etcd encryption only if you specifically require it for your use case.
```
per [IBM guidance on tense](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense), I understand this is existing text